src/components/Staking/StakingDashboard.tsx

```diff
@@ -127,15 +128,17 @@ export function StakingDashboard() {
           );
 
           const amount = delegation?.balance?.amount || "0";
-          const formattedAmount = (parseInt(amount) / 1000000).toLocaleString();
+          const formattedAmount = (
+            parseInt(amount) / 1000000000000000000
```

Comment:
We have this decimal converter in multiple place, this should be a constant and we should have a util convert function